### DISCUSSION TITLE
Code not highlighted in edit view

### DISCUSSION THREAD

**Discussion by BernieHuang2008** [community/community#75368] [Mobile]
When editing a file, the code won't be highlighted, so it's confusing and a mess.

![Screenshot_20231113_212733_com.github.android.jpg](https://github.com/community/community/assets/88757735/8c4ca5a0-0f4b-42e8-ab40-1feb00a5887f)

**Comment by HenSquared** [comment]
Thanks for the feedback. Just to make sure we have a file we can test with, can you share a URL to a publicly accessibly file that reproduces this issue? Thank you. 

In the meantime, I've opened up an issue with the Android team to take a look. 
Thanks.

**Reply by BernieHuang2008** [reply]
I just thought it's a common problem??

but of course. This URL below is the file in screenshot.

https://github.com/BernieHuang2008/PySQLite/blob/main/gensql.py

For more information, Version `v1.137.0(755)` is what I'm currently using.

**Reply by HenSquared** [reply]
My mistake @BernieHuang2008 , thanks for the additional information. I've opened a feature request for this. Thanks!
